Simulator/player.py

```diff
@@ -1108,4 +1131,32 @@ def remove_from_pool(self, item):
     def random_item_from_pool(self):
         item = random.choice(self.item_pool)
         self.remove_from_pool(item)
-        return item
\ No newline at end of file
+        return item
+
+    @property

```

Comment:
@silverlight6 Found a great way to use class variables and still don't need to generate the full vector